staging/src/k8s.io/apiserver/pkg/admission/plugin/cel/validator.go

```diff
@@ -198,17 +203,16 @@ func (v *CELValidator) Validate(versionedAttr *generic.VersionedAttributes, vers
 		request:   requestVal.Object,
 	}
```

Comment:
Probably not for this PR, but there's an optimization opportunity here to reuse a lot of these values through the whole admission step for a given request.

Maybe just add a TODO?